airflow/dags/gtfs_static_downloader.py

```diff
@@ -22,22 +24,23 @@ def make_gtfs_list():
     kwargs:
      catalog = a intake catalog containing an "official_list" item.
     """
-    df = pd.read_csv(
-        (
-            "https://docs.google.com/spreadsheets/d/"
-            "1qr49azk6p30mp96_7myKoO-Bb_bXMMn5ZzgbL-uPiPw/gviz/"
-            "tq?tqx=out:csv&sheet=Data"
-        ),
-        usecols=["ITP_ID", "GTFS", "Agency Name"],
-        dtype={"ITP_ID": "float64"},
-    )
+
+    agencies = yaml.safe_load(open("data/agencies.yml"))
+
+    # yaml has form <agency_name>: { agency_name: "", gtfs_schedule_url: [...,] }
+    df = pd.DataFrame.from_dict(agencies, orient="index")
+
+    # TODO: handle multiple urls
+    # currently stores urls as a list, so get first (and hopefully only) entry
+    df["gtfs_schedule_url"] = df["gtfs_schedule_url"].str.get(0)
+
     # TODO: Figure out what to do with Metro
     # For now, we just take the bus.
 
     # TODO: Replace URLs with Zip ones.
     # For now we filter, and then remove ones that don't contain
     # zip filters
-    df = df[(df.GTFS.str.contains("zip")) & (df.GTFS.notnull())]
+    df = df[(df.gtfs_schedule_url.str.contains("zip")) & (df.gtfs_schedule_url.notna())]
```

Comment:
we should probably go through the list and identify which ones are not returning zip files and replacing the urls as need.

Reply:
For example, Glendale Beeline doesn't have `.zip` in the URL but does successfully return a zip file when you go to the following url http://glendaleca.gov/Home/ShowDocument?id=29549

Reply:
That makes sense--if it works for you, I can create a quick notebook tomorrow that checks the urls and chuck it on the notebooks repo!

Reply:
perf! Also adding to the agenda for the MSD meeting at 2:30 (check your calendar for an invite!)